neo/Prompt/Commands/Invoke.py

```diff
@@ -336,19 +336,18 @@ def test_invoke(script, wallet, outputs, withdrawal_tx=None,
             wallet_tx.Gas = tx_gas
             # reset the wallet outputs
             wallet_tx.outputs = outputs
-            wallet_tx.Attributes = []
+            wallet_tx.Attributes = [] if invoke_attrs is None else invoke_attrs
```

Comment:
Currently I'm only re-applying the attributes that were passed to the function initially. I'm a little bit in doubt whether we should also restore one of the following attributes.

this: https://github.com/ixje/neo-python/blob/99adf951a4798f344fef02acd884f6a7ee0edac8/neo/Prompt/Commands/Invoke.py#L268
or this one
https://github.com/ixje/neo-python/blob/99adf951a4798f344fef02acd884f6a7ee0edac8/neo/Prompt/Commands/Invoke.py#L284

feedback appreciated why we should or should not.